### ISSUE TITLE
Add possibility to add persons names of not registered persons

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1247]
There are at least two cases (that does not ocrrur very often but sometimes) for this:

1. Services provided by Persons. At the moment we give the service a location and skip the person. This is nor fair to these people also if we mention them in the description or some other place.

2. Storys from persons not registered to the map republished on the map (see also issue #912) 

I suggest that it should be possible to ad persons as free text in both cases.

**Comment by literarymachine** [comment]
> Services provided by Persons. At the moment we give the service a location and skip the person. This is nor fair to these people also if we mention them in the description or some other place.

Why not just contact the people and ask if they want to register to be listed as the provider of the service?

> Storys from persons not registered to the map republished on the map (see also issue #912)

As @acka47 says in https://github.com/hbz/oerworldmap/issues/912#issuecomment-308352289, you can add a free text citation for republications (currently only on beta) and include an author there. If the publication is original, why not ask the author to register to be listed as the author of the story?

**Comment by karindr** [comment]
Yes, we allways can aks these people, but there are two problems:

- what to do, in the case they don't want to register or simply don't do it
- when you are working on long lists (like at the moment) you have to do this work fast and you cant wait for people answering to mails. A way could be to add the peoples name and contact them as a second step.

**Comment by karindr** [comment]
There is a third case where we should be able to add peolpes names who are not registered:

- contact persons for organisations (their name published on their website)

And these are cases that will also happen to other people putting data to the map on projects etc. that are not their own for example country champions and other editors (like Katy).

**Comment by literarymachine** [comment]
> There is a third case where we should be able to add peolpes names who are not registered:

I understand all of these, but we have (had?) the policy to not publish personal data of people without there consent, i.e. without them creating a profile. Is this not so any longer, @trugwaldsaenger?

**Comment by karindr** [comment]
I don't suggest to create profiles for people. I only suggest to mention their name like on wilikedia for example about serlo: "Im April 2009 wurde sie von dem Projektgründer Simon Köhl ..."

**Comment by acka47** [comment]
@karindr mentions three cases where the requested functionality is needed:

> 1. Services provided by Persons. At the moment we give the service a location and skip the person. This is nor fair to these people also if we mention them in the description or some other place.
> 2. Storys from persons not registered to the map republished on the map (see also issue #912) 
> 3. contact persons for organisations (their name published on their website)

Regarding 1.) I don't see that much a difference between mentioning the maintainer of a service in the description or in an extra field.

Regarding 2.), we already have the possibility to write the creator in the rights statement field (see https://github.com/hbz/oerworldmap/issues/1247#issuecomment-308386452 & https://github.com/hbz/oerworldmap/issues/912#issuecomment-308352289). And I think this information is already displayed quite prominently so that the original author gets the credit she deserves. (A bigger problem is adding another person as creator as I have mentioned in https://github.com/hbz/oerworldmap/issues/912#issuecomment-308360407.)

Regarding 3.), I think this is the most striking example in favor of adding personal names for people not on the world map. We could use a more general possibility for adding a contact point, either by chosing a person from the OER world map or by adding some information (email, telephone, name). THis would also come handy for collective email adresses or mailing lists.

**Comment by karindr** [comment]
Regarding 1: There is a difference since the provider is now also shown very prominent in the subtitle.

Regarding 2: Ok with the copyright line in the subtitle it is nearly the same. +1

**Comment by trugwaldsaenger** [comment]
> Services provided by Persons. At the moment we give the service a location and skip the person. This is nor fair to these people also if we mention them in the description or some other place.
> 
> Why not just contact the people and ask if they want to register to be listed as the provider of the service?

This certainly increases the time needed for the entry. But since th problem does not occur very often, I think it would be worth giving it a try.

**Comment by trugwaldsaenger** [comment]
> There is a third case where we should be able to add peolpes names who are not registered:
> 
> I understand all of these, but we have (had?) the policy to not publish personal data of people without there consent, i.e. without them creating a profile. Is this not so any longer, @trugwaldsaenger?

Yes, we still have the policy not to create profiles for natural person. I`m not sure if just adding the name of a person to a service has to be treated differently from a legal perspective. I will clarify with our lawyer!

**Comment by trugwaldsaenger** [comment]
> Regarding 3.), I think this is the most striking example in favor of adding personal names for people not on the world map. We could use a more general possibility for adding a contact point, either by chosing a person from the OER world map or by adding some information (email, telephone, name). THis would also come handy for collective email adresses or mailing lists.

I agree. I think this example is more relevant from a practical point of view than the initial service/provider example.

**Comment by trugwaldsaenger** [comment]
closed by accident

**Comment by literarymachine** [comment]
Against our privacy policy.

**Comment by programmieraffe** [comment]
Talked about this with @trugwaldsaenger again - when we want to map OER actitivies it should be possible to link to non-registered persons, e.g. create a person entry for a researcher. It's similiar to create a wikipedia page about a relevant person. 

Wikidata has some guidelines regarding the privacy issues:
https://www.wikidata.org/wiki/Wikidata:Living_people

Also related to #294

**Comment by literarymachine** [comment]
> it should be possible to link to non-registered persons, e.g. create a person entry for a researcher. 

Ok - no big issue from the technical perspective!
